.gitignore

```diff
@@ -8,6 +8,7 @@ node_modules
 .pnp.loader.mjs
 static/dist
 .env
+.wrangler/
```

Comment:
You sure about this?

Reply:
Yes, checkout https://github.com/ubiquity/pay.ubq.fi/blob/development/.gitignore

Reply:
Just because it's there doesn't mean it's correct

Reply:
I wasn't inspired by it, the files in .wrangler are either temporary or state related. Temp files are generated at run time, we also don't want to share kv state so it makes sense to ignore it.

![image](https://github.com/user-attachments/assets/1bbdf001-e12d-4675-a895-ce4c0bc0b7b8)